dlrs/clearlinux/tensorflow_2/mkl/README.md

```diff
@@ -1,27 +1,26 @@
 ## Deep Learning Reference Stack with TensorFlow 2.0 and Intel® MKL-DNN
 
-[![](https://images.microbadger.com/badges/image/clearlinux/stacks-dlrs_2-mkl.svg)](https://microbadger.com/images/clearlinux/stacks-dlrs_2-mkl "Get your own image badge on microbadger.com")
-[![](https://images.microbadger.com/badges/version/clearlinux/stacks-dlrs_2-mkl.svg)](https://microbadger.com/images/clearlinux/stacks-dlrs_2-mkl "Get your own version badge on microbadger.com")
+[![](https://images.microbadger.com/badges/image/clearlinux/stacks-dlrs_2-mkl.svg)](https://microbadger.com/images/clearlinux/stacks-dlrs_2-mkl "Get your own image badge on microbadger.com")	
```

Comment:
Ah, we have to update these badges to point to the new dockerhub namespace

Reply:
Update, does that look good?

Reply:
Please remove the rc0 tag